src/main/kotlin/no/nav/syfo/behandlerdialog/database/DialogmeldingOutQuery.kt

```diff
@@ -0,0 +1,73 @@
+package no.nav.syfo.behandlerdialog.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.domain.PersonIdent
+import java.sql.Connection
+import java.sql.ResultSet
+import java.sql.SQLException
+import java.time.OffsetDateTime
+import java.util.*
+
+const val queryGetDialogmeldingOutForPersonident =
+    """
+        SELECT *
+        FROM DIALOGMELDING_OUT
+        WHERE personident = ?
+    """
+
+fun DatabaseInterface.getDialogmeldingOutForPersonident(
```

Comment:
Burde det være `byPersonIdent` i stedet for `forPersonIdent`? Eller enda bedre: Bare `getDialogmeldingOut`, siden man uansett sender inn en PersonIdent?

Reply:
![image](https://user-images.githubusercontent.com/40055758/221525192-d6fea757-2a6a-4463-8aac-00a8ea665762.png)

Reply:
Nå skjønte jeg den! Get Out